### DISCUSSION TITLE
Default mode to greylist instead of blocked?

### DISCUSSION THREAD

**Discussion by appyface** [Cookie-AutoDelete/Cookie-AutoDelete#1431] [General] [answered]
Is it possible to have all sites (without expressions) treated as if they are greylisted, instead of as blocked, by default?

**Comment by kennethtran93** [comment, marked as answer]
Unfortunately not. 

The only way to treat any site as greylist default is to create a greylist expression with an asterisk `*` in it.

**Comment by appyface** [comment]
If I put an expression that is high level, like ^.*$ (or something like that?), will more granular expressions take precedence?

**Reply by kennethtran93** [reply]
Simply putting `*` in the greylist will suffice for the expression to match anything not whitelisted. Only domains found/matched in the whitelist will take precedence over greylist. There isn't a sort order for expressions inside a list yet, so it will only take the first one found.

**Comment by appyface** [comment]
Thanks so much, I will give this a try.
